docs/conf.py

```diff
@@ -377,9 +385,7 @@ def linkcode_resolve(domain, info):
 	if domain != "py" or not info["module"]:
 		return None
 	filename = info["module"].replace(".", "/")
-	theResult = str("{prefix}/blob/{branch}/{file}.py").format(
-		prefix=linkcode_url_prefix, branch="stable", file=filename
-	)
+	theResult = f"{linkcode_url_prefix}/blob/{DOCS_BUILD_REF}/{filename}.py"
```

Comment:
🤔 LGTM! Around line `388` the `linkcode_resolve` function now accounts for the branch or commit hash given by the new variable `DOCS_BUILD_REF`. Further the refactoring with the f-string syntax is consistent with recent project goals to modernize.